README.md

```diff
@@ -1,16 +1,10 @@
 # Media Dubiety
-Rewrite of Dispenser's NewbieAudioVideo IRC bot for T129845
+**Media Dubiety** is an IRC bot in channels defined in `~tools.media-dubiety/channels.json`. It sends messages such as:
```

Comment:
`~tools.media-dubiety/channels.json` applies for this instance of the running bot.